common/types/types.go

```diff
@@ -87,6 +87,13 @@ type LocalUser struct {
 //                    account to communicate with LDAP/AD. Hence this account
 //                    must have appropriate privileges, specifically for lookup.
 //  ServiceAccountPassword: of the service account
+//  StartTLS: if set, the connection will be upgrated to SSL/TLS mode
+//  InsecureSkipVerify: if set, the certificate verification is skipped;
+//                      used only when `StartTLS` is enabled.
+//  TLSCertIssuedTo: Servername for which the TLS/SSL certificate was issued.
+//                   if this is empty, the certs will not be verified and
```

Comment:
This comment isn't correct anymore.  The server is only vulnerable to MITM if `TLSCertIssuedTo` is empty *and* `InsecureSkipVerify` is explicitly enabled